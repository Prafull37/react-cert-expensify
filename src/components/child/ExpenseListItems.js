import React from 'react';
import {Link} from 'react-router-dom';
import moment from "moment";
import numeral from "numeral";

const ExpenseListItems=(props)=>(<div>
   <Link to={`/edit/${props.id}`}>
        <h6>{props.description}</h6>
        </Link>
        
        <div>{`â‚¹ ${numeral(props.amount/100).format('0,0.00')}`} - {moment(props.createdAt).format("DD/MM/YYYY")}</div>
        
    </div>
)

export default ExpenseListItems;