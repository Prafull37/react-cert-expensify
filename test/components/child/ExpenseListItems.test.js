import React from "react"
import {shallow} from "enzyme";
import ExpenseListItems from "../../../src/components/child/ExpenseListItems";
import expenses from "../../fixtures/expense";

test("should render it ",()=>{
    const wrapper=shallow(<ExpenseListItems {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot()
})