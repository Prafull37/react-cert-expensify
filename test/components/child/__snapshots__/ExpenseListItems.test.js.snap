// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render it  1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h6>
      Gum
    </h6>
  </Link>
  <div>
    â‚¹ 1.95
     - 
    01/01/1970
  </div>
</div>
`;
